lib/size-lg.css

```diff
@@ -38,68 +38,71 @@
    * Intentional redundancy build into each set of unit classes.
    * Supports: 2, 3, 4, 5, 6, 8, 10, 12 part
    *
-   * Uses `flex-basis: auto` with a width to avoid box-sizing bug in IE10/11 - http://git.io/vllMD
+   * Use `flex-basis: auto` with a width to avoid box-sizing bug in IE10/11
+   * http://git.io/vllMD
    */
 
-  .u-lg-size1of12 {
+  /* postcss-bem-linter: ignore */
+  [class*="u-lg-size1"],
```

Comment:
Seems this effects the sizeFit and Full rules as well. Going to revert to being more explicit as before